encoder/libav_encoder.cpp

```diff
@@ -279,8 +292,8 @@ void LibAvEncoder::initAudioOutCodec(VideoOptions const *options, StreamInfo con
 LibAvEncoder::LibAvEncoder(VideoOptions const *options, StreamInfo const &info)
 	: Encoder(options), output_ready_(false), abort_video_(false), abort_audio_(false), video_start_ts_(0),
 	  audio_samples_(0), in_fmt_ctx_(nullptr), out_fmt_ctx_(nullptr), segment_start_ts_(0), segment_num_(0),
-	  virtual_video_ts_(0), virtual_audio_ts_(0), previous_video_timestamp_(0), previous_audio_timestamp_(0),
-	  feed_encoder_frames_(true), previous_feed_value_(true)
+	  previous_video_timestamp_(0), pause_timestamp_(-100000), unpause_timestamp_(-1000000), pausestate_(false),
```

Comment:
Why -100000?  I think 0 would be preferred to be consistent with the rest of the init values.